client-app/src/Components/DonorForm.tsx

```diff
@@ -2,6 +2,294 @@ import React, { useState, ChangeEvent, FormEvent } from 'react';
 import axios from 'axios';
 import '../css/DonorForm.css';
 
+interface FormData {
+    itemType: string;
+    currentStatus: string;
+    donorEmail: string;
+    program: string;
+    imageUpload: string[];
```

Comment:
Image Upload is stored as a string array[], but it should have been stored as a file upload.